doc/CHEATSHEET.md

```diff
@@ -243,6 +244,64 @@ txSkelTemplate
     }
 ```
 
+### Attach a Proposal Procedure to a transaction
+
+* Using the builtin constructor for proposals.
+
+```haskell
+txSkelTemplate
+  { ...
+    txSkelProposals =
+      [ TxSkelProposal
+          { txSkelProposalAddress = walletAddress (wallet 1),
+            txSkelProposalAction =
+              TxGovActionTreasuryWithdrawals $
+                Map.fromList
+                  [ (toCredential $ wallet 1, Api.Lovelace 100),
+                    (toCredential $ wallet 2, Api.Lovelace 10_000)
+                  ],
+            txSkelProposalWitness = (toScript myScript, myRedeemer),
+            txSkelProposalAnchor = Nothing
+          }
+      ]
+    ...
+  }
+```
+
+* Using smart constructors and (optional) helpers.
+
+```haskell 
+txSkelTemplate
+  { ...
+    txSkelProposals =
+      [ simpleTxSkelProposal
+          (wallet 1)
+          (TxGovActionParameterChange [FeePerByte 100, FeeFixed 1_000])
+          `withWitness` (myScript, myRedeemer)
+          `withAnchor` "https://www.tweag.io/"
+      ]
+    ...
+  } 
+```
+
+### Anchor resolution policy
+
+* Auto resolution using Http (very unsafe, block reproducibility)
+
+```haskell 
+    txSkelOpts = def 
+	  { txOptAnchorResolution = AnchorResolutionHttp
+	  }
+```	
+
+* Auto resolution using a given map with resolved page content as bytestrings.
```

Comment:
Maybe this should be first in the list since it is the default and recommended way